src/Core/WinSWCore/WinSWCore.csproj

```diff
@@ -73,6 +73,7 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />
+    <None Include="winsw_key.snk" />
```

Comment:
The SNK must not be included into the repo

Reply:
Done